addons/portal/views/mail_followers_templates.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="message_document_unfollowed" inherit_id="mail.message_document_unfollowed">
```

Comment:
:socks:  Shouldn't the file be "mail_templates_public.xml" ?

Reply:
Yes indeed, that’s more logical.